cirq/google/sim/xmon_simulator.py

```diff
@@ -53,6 +54,37 @@
 from cirq.study import ParamResolver, Sweep, Sweepable, TrialResult
 
 
+def pretty_state(wvf, decimals=2):
```

Comment:
since we call the object passed in here a state of final_state, I'd use state instead of wft